lib/portage/gpkg.py

```diff
@@ -1847,11 +1847,12 @@ def _check_pre_image_files(self, root_dir, image_prefix="image"):
             image_total_size,
         )
 
-    def _check_pre_quickpkg_files(self, contents, root):
+    def _check_pre_quickpkg_files(self, contents, root, image_prefix="image"):
```

Comment:
When would the prefix not be "image"?

Reply:
None at the time. I just do that so easier to do extension.

Reply:
Alrighty.